hub-config/tasks.json

```diff
@@ -8,17 +8,165 @@
+                                "VA",
+                                "WA",
+                                "WV",
+                                "WI",
+                                "WY",
+                                "PR"
+                            ]
+                        },
+                        "variant": {
+                            "required": ["JN.1", "JN.1.7", "JN.1.8"],
+                            "optional": null
+                        },
+                        "horizon": {
+                            "required": null,
+                            "optional": [
+                                -27,
+                                -26,
+                                -25,
+                                -24,
+                                -23,
+                                -22,
+                                -21,
+                                -20,
+                                -19,
+                                -18,
+                                -17,
+                                -16,
+                                -15,
+                                -16,
+                                -15,
+                                -14,
+                                -13,
+                                -12,
+                                -11,
+                                -10,
+                                -9,
+                                -8,
+                                -7,
+                                -6,
+                                -5,
+                                -4,
+                                -3,
+                                -2,
+                                -1,
+                                0,
+                                1,
+                                2,
+                                3,
+                                4,
+                                5,
+                                6,
+                                7 
+                            ]
                         }
                     },
-                    "output_type": {},
-                    "target_metadata":[]
+                    "output_type": {
+                        "mean": {
+                            "output_type_id": {
+                                "required": ["NA"],
+                                "optional": null
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        },
+                        "sample": {
+                            "output_type_id_params": {
+                                "is_required": false,
+                                "type": "character",
+                                "max_length": 500,
+                                "min_samples_per_task": 96,
```

Comment:
There has been discussion about this elsewhere, but I would advocate for this being set to 100, just to force people to generate 100 samples.